modules/setting/setting.go

```diff
@@ -1372,6 +1374,11 @@ func newMailService() {
 	MailService.FromName = parsed.Name
 	MailService.FromEmail = parsed.Address
 
+	MailService.SendmailArgs, err = shellquote.Split(sec.Key("SENDMAIL_ARGS").String())
```

Comment:
Might be it's worth load this (and also parse ) only if `MailService.UseSendmail` is enabled

Reply:
Done